python/ambassador/config/config.py

```diff
@@ -234,6 +234,14 @@ def good_ambassador_id(self, resource: dict):
         # Is an ambassador_id present in this object?
         allowed_ids: StringOrList = resource.get('ambassador_id', 'default')
 
+        # If we find the array [ '_automatic_' ] then allow it, so that hardcoded resources
+        # can have a useful effect. This is mostly for init-config, but could be used for
+        # other things, too.
+
+        if allowed_ids == [ "_automatic_" ]:
+            self.logger.debug(f"ambassador_id {allowed_ids} always accepted")
+            return True
+
```

Comment:
(to type words that I said to Flynn in-person):

Given that this is strict (compared to say `if "_automatic_" in allowed_ids:`) because it's a hack, and isn't something that we want users to think they can type: Would it be better to have the `_automatic_` namespace logic also munge the resource's `ambassador_id`?  This would have the advantage that a user literally _can't_ type it; `_` is illegal in a real namespace.